infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -43,39 +35,70 @@ versioned_builds = [
   # Remove libtpu from PyPI builds
   {
     git_tag         = "v2.2.0-rc1"
-    package_version = "2.2.0-rc1"
+    package_version = "2.2.0rc1"
```

Comment:
we could keep rc1 now, and we could change quick to rc4 since PyTorch has v2.2.0-rc4 tag too

Reply:
I think just tag rc5 (or whatever the current tag is when you read this message). It looks like upstream created multiple RC tags within the past 24 hours -- the early ones probably just had mistakes in them that they caught.

Reply:
I am confused about which version to use. We have v2.2.0-rc1 now and pytorch have v2.2.0-rc5 already. Which version should we use for:
`git_tag` and `pytorch_git_rev` and `package_version`?

Reply:
should we use `rc1` currently? cc @will-cromar for advice again after online chet discuss

for 2.2 release, they have `torch-2.2.0.dev20231206%2Bcpu.cxx11.abi-cp311-cp311-linux_x86_64.whl` and `torch-2.2.0%2Bcpu.cxx11.abi-cp311-cp311-linux_x86_64.whl` at the same time, formal release would only have the latter

following is local test results:
- torch installed by `pip install --user --pre --no-deps torch torchvision --extra-index-url https://download.pytorch.org/whl/test/cpu` is at commit `24397727a8500e2ec598a5ae3efb72d153da9b64` merged at Dec6

- torch installed by `pip install https://download.pytorch.org/whl/test/cpu-cxx11-abi/torch-2.2.0.dev20231206%2Bcpu.cxx11.abi-cp311-cp311-linux_x86_64.whl` is at commit `54aca571d6ded30bc5e6e19051bd460db6a631d4` merged at Dec6 early

- torch installed by `pip install https://download.pytorch.org/whl/test/cpu-cxx11-abi/torch-2.2.0%2Bcpu.cxx11.abi-cp311-cp311-linux_x86_64.whl` is built with commit `12bcfddce5773410d6e859e3c0feda9dc6c9b8d3` merged at Dec7 early, there is newer commit than this one at Dec7 too

since there is no formal announce for rc in blog now, shall we use rc1?

Reply:
for `git_tag and pytorch_git_rev and package_version`, if we use rc1, we would use:
```
git_tag         = "v2.2.0"
pytorch_git_rev = "v2.2.0-rc1"
package_version = "2.2.0rc1"
```

Reply:
OK, updated git_tag.